src/Illuminate/Support/Carbon.php

```diff
@@ -6,5 +6,27 @@
 
 class Carbon extends BaseCarbon
 {
-    //
```

Comment:
Please avoid to create gaps between Laravel Carbon and the original Carbon class. For example if someone here want to use CarbonImmutable, then, he loose your feature.

You should first propose the change to the Carbon project in a standard way that would fit not only for Laravel but for any PHP app or other framework that can have this problematic.